### ISSUE TITLE
Couldn't hide border of the last child from thead

### ISSUE THREAD

**Issue by 18601673727** [pupudu/window-table#26]
Hi, 
Thanks for bring us such a nice lib! Really helpful but there seems to be a tiny flaw:
```css
.table-header-cell, .table-cell {
    padding: 15px;
    border-bottom: 4px solid var(--light);
    border-right: 4px solid var(--light);

    &:last-child {
      border-right: 0;
    }
}
```
Result:
![image](https://user-images.githubusercontent.com/3302620/62254023-46be1800-b42a-11e9-9106-2934234bbf05.png)

**Comment by pupudu** [comment]
Hi there,
Thank you for creating this issue. 

Could you tell me a bit more about this issue? Does it happen only for the text columns? 

Or is it possible for you to reproduce this in a demo codesandbox or something? Would be pretty easy to fix it if you can do that.

Cheers.

**Comment by 18601673727** [comment]
Hi @pupudu, 

Of course, here you go: https://codesandbox.io/embed/thirsty-dewdney-u3pro

**Comment by pupudu** [comment]
Hi @18601673727 
This issue has been fixed with v0.8.11

Thanks again for opening this issue
